graphrag/query/context_builder/community_context.py

```diff
@@ -140,7 +140,12 @@ def _cut_batch() -> None:
         if len(record_df) == 0:
             return
         current_context_text = record_df.to_csv(index=False, sep=column_delimiter)
-        all_context_text.append(current_context_text)
+        if (not all_context_text) and single_batch:
+            all_context_text.append(
+                f"-----{context_name}-----" + "\n" + current_context_text
```

Comment:
if using f-string you could do `f"-----{context_name}-----\n{current_context_text}"`

Reply:
Updated